release-content/0.14/release-notes/11989_Implement_the_AnimationGraph_allowing_for_multiple_animati.md

```diff
@@ -1 +1,58 @@
-TODO
+Through the eyes of a beginner, handling animation seems simple enough.
+Define a series of keyframes, transform, rotate and scale the various bits of your model to match those poses.
+The user tells you when to start and stop an animation, we slap some interpolation on there to smoothly move between them, and you're done!
+
+But modern animation pipelines (especially in 3D!) are substantially more complex:
+animators expect to be able to smoothly blend and programmatically alter different animations dynamically in response to gameplay.
+In order to capture this richness, the industry has developed the notion of an **animation graph**, which is used to couple the underlying [state machine] of a game object to the animations that should be playing, and the transitions that should occur between each of the various states.
+
+A player character may be walking, running, slashing a sword, defending with a sword...
+To create a polished effect, animators need to be able to change between these animations smoothly, change the speed of the walk cycle to match the movement speed along the ground and even perform multiple animations at once!
+
+In Bevy 0.14, we've implemented the [Animation Composition RFC], providing a low-level API that brings code- and asset-driven animation blending to Bevy.
+
+```rust
+#[derive(Resource)]
+struct ExampleAnimationGraph(Handle<AnimationGraph>);
+
+fn programmatic_animation_graph(
+    mut commands: Commands,
+    asset_server: ResMut<AssetServer>,
+    animation_graphs: ResMut<Assets<AnimationGraph>>,
+) {
+    // Create the nodes.
+    let mut animation_graph = AnimationGraph::new();
+    let blend_node = animation_graph.add_blend(0.5, animation_graph.root);
+    animation_graph.add_clip(
+        asset_server.load(GltfAssetLabel::Animation(0).from_asset("models/animated/Fox.glb")),
```

Comment:
I'm not familiar with animation at all. Would it make sense to assign every animation to a named variable first? running_animation, jumping_animation, etc. and combine them later in the graph? It might make more sense that way for the unfamiliar reader.